ingest/monitor.py

```diff
@@ -47,6 +47,7 @@ def setup_logger(logger_name, log_file, level=logging.DEBUG, format="default"):
 def log_exception(dev_logger, user_logger, exception):
     user_logger.critical(str(exception))
     dev_logger.exception(exception)
+    print(str(exception))
```

Comment:
I assume this isn't intended, so:

```suggestion
```

Reply:
It's a PITA to have to visit a log file to figure out what caused a validation to fail... Would it be acceptable to enclose it in an "if bypass_mongo_writes():" conditional? It seems odd to import config into the monitor module to do so though :(

Reply:
I'm personally fine with this, it's just a bit unconventional for us to print production stuff to the console in JS and Python.

So either your suggested conditional, a brief code comment, or merely your explanation above seem sufficient to me.

Reply:
Removing for now (importing config to enable the conditional would require testing so I'll consider adding it back in my next batch of work)